Curso-Vue-2024/indecision-app
# Testing: test on Vue components

Testing: test on Vue components

* Create MyCounter.test.ts.
* Write test 'should match snapshot'.
* Create vitest.config.ts
